recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "harmonica" %}
-{% set version = "0.1.0" %}
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 552fb15217de85736dc7fea045feccc4c5a4bec3d6b25e617124f6fd633c288f
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-1.tar.gz
+  sha256:
```

Comment:
We need a checksum here. Also I'm not sure that ver-1 is a pep440 valid version. Maybe we should raise that upstream.